src/components/DomEvents.js

```diff
@@ -180,12 +207,17 @@ function DomEvents() {
       <div className="editor md:h-56 flex-auto overflow-hidden">
         <div className="h-56 md:h-full w-full flex flex-col">
           <div className="h-8 flex items-center w-full text-sm font-bold">
-            <div className="p-2 w-16">#</div>
+            <div
+              className="p-2 w-16 cursor-pointer"
+              onClick={changeSortDirection}
+            >
+              # {getSortIcon('id')}
```

Comment:
'id' not necessary

Reply:
Yes you're right it was of the previous implementation. Thanks :)

Reply:
Perfect, tks! I noticed it while reading your code. Good job here.